.travis.yml

```diff
@@ -18,12 +18,12 @@ install:
   - bundle install --retry=3
 
 rvm:
-  - 1.9
-  - 2.0
+  - 1.9.3
```

Comment:
I actually moved this change into https://github.com/YorickPeterse/oga/pull/157.